sys/net/crosslayer/ng_pktbuf_static/ng_pktbuf_static.c

```diff
@@ -429,6 +429,8 @@ static inline bool _too_small_hole(_unused_t *a, _unused_t *b)
 
 static inline _unused_t *_merge(_unused_t *a, _unused_t *b)
 {
+    assert(b->next != NULL);
```

Comment:
Okay… but this is a valid case, if `b` is at the end of the packet buffer (which is a pretty frequent case).